dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -73,6 +73,7 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
 
         try
         {
+            wil::winrt_module_reference moduleRef{};
```

Comment:
Might want to move it one level above the try catch. If developers add something outside the try catch for example, this would no longer apply.